### PR TITLE
upgrade to clojure 1.8.0

### PR DESCRIPTION
**Pull request by kgann**
@rbataev 

If this is good, can I just deploy to Clojars?

### CODE REVIEW

File: src/pallet/action.clj [hunk 1]
```diff
@@ -226,7 +226,7 @@ of user options.
         (IllegalArgumentException.
          (format
           "Attempting to implement action %s with invalid dispatch value %s"
-          (action-symbol action) dispatch-val)))))
+          (action-symbol action))))))
```
Comment (candid82) [inline src/pallet/action.clj]: Format string has two placeholders, so should not `format` take two aruments (besides the format string)?

Comment (kgann) [reply src/pallet/action.clj]: I blame Hugo and his crazy formatting. I thought this was another instance where `throw` passed two arguments. See 19e5662dc373c547a34dc8691efb09d790ca7909

**Comment by kgann** [comment]
Upon closer inspection, I believe the intent was 96b75faf65f850346dab571c44a709cd51c9a5f0

**Comment by candid82** [comment]
lgtm

**Pull request merged**